src/General/Heading/Heading.test.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+
+import * as renderer from 'react-test-renderer';
+import '@testing-library/jest-dom/extend-expect';
+import { render } from '@testing-library/react';
+
+import Heading from './Heading';
+
+const heading = 'This is a Heading';
+
+it(`<Heading> should match snapshot when the component is called`, () => {
+  const HeadingSnapshot = renderer
+    .create(<Heading>{heading}</Heading>)
```

Comment:
can add classname prop and see whether the classname is added to the correct element.